test/pspm_test_temp.m

```diff
@@ -0,0 +1,122 @@
+function pspm_test_temp(varargin)
```

Comment:
This is a function for temporary testing only selected functions. It will be removed when merging it to the develop branch.